src/iqstat.cpp

```diff
@@ -357,3 +227,116 @@ void usage() {
     }
     printReleaseInfo( "iqstat" );
 }
+
+auto show_RuleExec( char *userName,
+                    const char *ruleName,
+                    int allFlag,
+                    bool brief ) -> int
+{
+    int status;
```

Comment:
Please move declaration to the point of initialization

Reply:
Done